src/solver/conv_asm_dir_BwdWrW3x3.cpp

```diff
@@ -38,7 +38,6 @@
 #include <miopen/generic_search.hpp>
 
 #define WORKAROUND_ISSUE_532 1 // ConvAsmBwdWrW3x3 has precision issues with some PerformanceConfigs
-#define WORKAROUND_ISSUE_1146 1 // check asm solver applicability for gfx90a
```

Comment:
Thanks! Put 1 in the can and 8 more to go :-)

Reply:
@junliume This PR shouldn't be necessary for that SWDEV.